src/btree/bt_vrfy.c

```diff
@@ -627,17 +621,7 @@ __verify_tree(
     case WT_PAGE_COL_FIX:
```

Comment:
We aren't removing `WT_PAGE_COL_FIX` in this ticket, which is why I kept this.

Reply:
[5335249](https://github.com/wiredtiger/wiredtiger/pull/12814/commits/533524914522f73843c057a9e587de4455d5a335)